servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcRouter.java

```diff
@@ -196,18 +200,25 @@ RouteProviders drainRoutes() {
             return new RouteProviders(allRoutes);
         }
 
+        GrpcExecutionStrategy alreadyRegisteredExecutionStrategy(final String path,
```

Comment:
Rename `alreadyRegisteredExecutionStrategy` => `executionStrategyFor()`

It does not seem like "already registered" is providing any additional value here.